src/RwaSpell.sol

```diff
@@ -0,0 +1,235 @@
+        ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+
+    address constant MCD_VAT      = 0xbA987bDB501d131f766fEe8180Da5d81b34b69d9;
+    address constant MCD_CAT      = 0xdDb5F7A3A5558b9a6a1f3382BD75E2268d1c6958;
+    address constant MCD_JUG      = 0xcbB7718c9F39d05aEEDE1c472ca8Bf804b2f1EaD;
+    address constant MCD_SPOT     = 0x3a042de6413eDB15F2784f2f97cC68C7E9750b2D;
+    address constant MCD_END      = 0x24728AcF2E2C403F5d2db4Df6834B8998e56aA5F;
+    address constant ILK_REGISTRY = 0xedE45A0522CA19e979e217064629778d6Cc2d9Ea;
+
+    /*
+        OPERATOR: 0xD23beB204328D7337e3d2Fb9F150501fDC633B0e
+        TRUST1: 0xda0fab060e6cc7b1C0AA105d29Bd50D71f036711
+        TRUST2: 0xDA0111100cb6080b43926253AB88bE719C60Be13
+        ILK: RWA001-A
+        RWA001: 0x9D7F8D3332a460344C1FC34624A4fB0B9d2fB2eE
+        MCD_JOIN_RWA001_A: 0xFeaa20404EF114BDC4a8d667dACc2A2CD87b0E63
+        MCD_FLIP_RWA001_A: 0x28749c007cd3D0fb67Db80682d6E3A9E25CC98c9
+        RWA001_A_URN: 0x10b7890081AEab7fA866be1A0314024EDe851f68
+        RWA001_A_CONDUIT_IN: 0xa1da5fa4920E5926126b5088B9Ce2321e6113812
+        RWA001_A_CONDUIT_OUT: 0x6826Db7A8CfE9709baC20345A0e7be40B251bFfB
+        RWA001_LIQUIDATION_ORACLE: 0x001c86aD3feF5b7CA6CC09f96d678bA060E5Cb61
+    */
+    address constant RWA001_OPERATOR           = 0xF00DBabEf00DbAbEF00DbabEF00DBABef00dbAbE;
+    address constant RWA001_GEM                = 0x9D7F8D3332a460344C1FC34624A4fB0B9d2fB2eE;
+    address constant MCD_JOIN_RWA001_A         = 0xFeaa20404EF114BDC4a8d667dACc2A2CD87b0E63;
+    address constant MCD_FLIP_RWA001_A         = 0x28749c007cd3D0fb67Db80682d6E3A9E25CC98c9;
+    address constant RWA001_A_URN              = 0x10b7890081AEab7fA866be1A0314024EDe851f68;
+    address constant RWA001_A_CONDUIT_IN       = 0xa1da5fa4920E5926126b5088B9Ce2321e6113812;
+    address constant RWA001_A_CONDUIT_OUT      = 0x6826Db7A8CfE9709baC20345A0e7be40B251bFfB;
+    address constant RWA001_LIQUIDATION_ORACLE = 0x001c86aD3feF5b7CA6CC09f96d678bA060E5Cb61;
+
+    uint256 constant SIX_PCT_RATE    = 1000000001847694957439350562;
+
+    // precision
+    uint256 constant public THOUSAND = 10 ** 3;
+    uint256 constant public MILLION  = 10 ** 6;
+    uint256 constant public WAD      = 10 ** 18;
+    uint256 constant public RAY      = 10 ** 27;
+    uint256 constant public RAD      = 10 ** 45;
+
+    uint256 constant RWA001_A_INITIAL_DC    = 1000 * RAD;
+    uint256 constant RWA001_A_INITIAL_PRICE = 1060 * WAD;
+
+    function execute() external {
+        // RWA001-A collateral deploy
+
+        // Set ilk bytes32 variable
+        bytes32 ilk = "RWA001-A";
+
+        // add RWA-001 contract to the changelog
+        CHANGELOG.setAddress("RWA001", RWA001_GEM);
+        CHANGELOG.setAddress("MCD_JOIN_RWA001_A", MCD_JOIN_RWA001_A);
+        CHANGELOG.setAddress("MCD_FLIP_RWA001_A", MCD_FLIP_RWA001_A);
+        CHANGELOG.setAddress("RWA001_LIQUIDATION_ORACLE", RWA001_LIQUIDATION_ORACLE);
```

Comment:
I think the LIQUIDATION_ORACLE is re-usable. Should it be `RWA001_*`?

Reply:
Or `MIP21_LIQUIDATION_ORACLE`

Reply:
Or `RWA_LIQUIDATION_ORACLE`

Reply:
Good point, perhaps it should be renamed to `MIP21_LIQUIDATION_ORACLE` here.